### PR TITLE
feat: Adding rg, vnet, container registry

### PR DESCRIPTION
**Pull request by pamelaluquemck**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,4 +1,21 @@
-resource "azurerm_resource_group" "default_rg" {
-  name     = "default-resource-group"
-  location = "eastus"
+# resource group
+resource "azurerm_resource_group" "az-capabilities-rg" {
```
Comment (jose-sal-y-rosas) [inline main.tf:2]: Why do we need to create a resource group? Please us the specific resource group for the project.

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

File: main.tf (hunk 1 above)
Comment (pamelaluquemck) [reply main.tf:2]: There is no resource group in the subscription 
image

File: main.tf [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+  address_space       = ["10.0.0.0/16"]
+}
+
+#azure container registry
+resource "azurerm_container_registry" "az-capabilities-acr" {
+  name                = "azcapabilitiesacr"
+  resource_group_name = "az-capabilities-rg"
+  location            = "eastus2"
+  admin_enabled       = true
+  sku                 = "Standard"
```
Comment (pramirezma) [inline main.tf:21]: Seems that there is no closing bracket for this resource

**Review by pramirezma — CHANGES_REQUESTED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

File: main.tf [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
-  name     = "default-resource-group"
-  location = "eastus"
+# resource group
+resource "azurerm_resource_group" "az-capabilities-rg" {
+  name     = "az-capabilities-rg"
+  location = "eastus2"
 }
+
+# virtual network
+module "az-capabilities-vnet-1" {
```
Comment (jose-sal-y-rosas) [inline main.tf:8]: I think this module could be titled more general, something like: module "network"

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

File: .trunk/trunk.yaml [hunk 4]
```diff
@@ -17,8 +17,6 @@ lint:
     - git-diff-check
     - markdownlint@0.37.0
     - prettier@3.0.3
-    - terrascan@1.18.3
```
Comment (jose-sal-y-rosas) [inline .trunk/trunk.yaml:20]: why do we need to remove this libraries? Are they not required?

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

File: .trunk/trunk.yaml (hunk 4 above)
Comment (pamelaluquemck) [reply .trunk/trunk.yaml:20]: We are using tfsec so terrascan it's not required as these do the same function

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Review by pramirezma — APPROVED** [review]

**Pull request merged**